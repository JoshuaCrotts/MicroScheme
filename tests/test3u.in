
// This tests just a basic program with a while loop and calls to printd
// and prints

int gvar = 12;

void main() {
    int i = 10;
    prints("T ");
    printd(-i);
    while (i > 0) {
        printd(i);
        i--;
    }
    prints("Blast off!");
}
