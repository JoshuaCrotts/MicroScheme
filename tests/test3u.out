T -1010987654321Blast off!
